content/fr/container-orchestration.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Orchestration de Conteneurs
+status: Completed
+category: Concept
+---
+
+## Ce que c'est
+
+L'orchestration de [Conteneurs](/container/) fait référence à la gestion et à l'automatisation du cycle de vie d'applications conteneurisées au sein d'environnements dynamiques.
+Réalisée à l'aide d'un orchestrateur de conteneurs (la plupart du temps, [Kubernetes](/kubernetes)), elle permet les déploiements, le passage à l'échelle (automatique), (l'auto-)remédiation et le monitoring.
+L'orchestration est une métaphore:
+L'outil d'orchestration dirige les conteneurs tel un chef d'orchestre, s'assurant que chaque conteneur (ou musicien) fait ce qu'il doit faire.
+
+## Problème auquel il répond
+
+Gérer des [microservices](/microservices), la sécurité, et la communication réseaux à grande échelle — et des [systèmes distribués](/distributed-systems) en général — est compliqué, pour ne pas dire impossible, manuellement.
```

Comment:
I would move "manuellement" just after "Gérer" --> "Gérer manuellement des microservices [...]", what do you think?